pysatSpaceWeather/instruments/methods/f107.py

```diff
@@ -117,9 +116,12 @@ def combine_f107(standard_inst, forecast_inst, start=None, stop=None):
     Notes
     -----
     Merging prioritizes the standard data, then the 45day data, and finally
-    the forecast data
+    the forecast data.
 
-    Will not attempt to download any missing data, but will load data
+    Will not attempt to download any missing data, but will load data.
```

Comment:
I don't quite understand this comment. If a user needs to download data where do they go?

Reply:
The user is providing instruments as stated above.  They don't need to load data into those instruments, but the files need to be available.  If the data can't be loaded, it's treated as missing.  If you find the wording confusing, can you suggest an alternative?  It seems clear to me.